.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 jobs:
-  build:
+  build-job:
```

Comment:
Curious: why `-job`?

Reply:
Was just maintaining Phil and Jose's formatting decision from their example. I have no strong feelings about this, actually — perhaps it's needlessly explicit?

Reply:
I don't remember intentionally adding `-job` to the example Jose and I made; must have pulled it in by accident. I agree it's not needed; just a bit redundant at worst, not a big deal.